lms/templates/courseware/courseware.html

```diff
@@ -73,7 +76,7 @@
   <script type="text/javascript" src="${static.url('js/vendor/codemirror-compressed.js')}"></script>
 
   <%static:js group='courseware'/>
-  <%include file="../discussion/_js_body_dependencies.html" />
+  <%include file="/mathjax_include.html" args="disable_fast_preview=disable_fast_preview"/>
```

Comment:
@andy-armstrong It's not just discussions that need MathJax. For example, the "Numerical Input with Hints and Feedback" and "Math Expression Input" problem types need it too. So I don't think we can remove it from the "Course" tab.